documentation/ADRs/010_evaluation_metrics.md

```diff
@@ -0,0 +1,55 @@
+# Evaluation Metrics
+
+
+| ADR Info            | Details            |
+|---------------------|--------------------|
+| Subject             | Evaluation metrics |
+| ADR Number          | 010                |
+| Status              | Accepted           |
```

Comment:
Lets set this to proposed and then make a PR on the acceptance status that we invite HH, Mike and Mihai to.